doc/tietomalli.md

```diff
@@ -1,26 +1,340 @@
 # Tietomalli
 
-Tässä kuvassa esitetään geoviitteen peruskäsitteet ja miten ne liittyvät toisiinsa. Käsitteistöä on tarkoituksella yksinkertaistettu ollakseen havainnollisempi ja luokkahierarkiat eivät vastaa 1-1 tietokantatauluja (nämä tarkemmin nähtävissä täältä: Tietokanta) tai koodin rakennetta vaan ennenkaikkea geoviitteen kannalta oleellista osaa niiden tietosisällöstä.
+Tässä kuvassa esitetään geoviitteen peruskäsitteet ja miten ne liittyvät toisiinsa. Käsitteistöä on tarkoituksella
+yksinkertaistettu ollakseen havainnollisempi ja luokkahierarkiat eivät vastaa 1-1 tietokantatauluja vaan ennenkaikkea
+käsitteistön kannalta oleellista osaa niiden tietosisällöstä.
 
 ![](images/data_flow.png)
 
 ## Tarkat geometriat
 
-Ratatieto saapuu järjestelmään pääasiassa tarkkoina 3D geometrioina (kuvassa oikealla), jotka on piirretty suunnitteluohjelmissa. Nämä kuvastavat tiettyä rataverkon osaa tietyn suunnitelman tai toteuman hetkellä, osat ovat päällekäisiä tai niiden välistä voi puuttua pätkiä. Lisäksi ne voivat olla määritelty eri koordinaatistoissa, joiden muunnos ei ole häviötöntä. Niistä ei siis voi yksiselitteisesti koostaa koko rataverkkoa. Geometrioiden tietomalli noudattaa läheisesti lähdeaineiston (LandXML) käsitteistöä.
+Ratatieto saapuu järjestelmään pääasiassa tarkkoina 3D geometrioina, jotka on piirretty suunnitteluohjelmissa. Nämä
+kuvastavat tiettyä rataverkon osaa tietyn suunnitelman tai toteuman hetkellä, osat ovat päällekäisiä tai niiden välistä
+voi puuttua pätkiä. Lisäksi ne voivat olla määritelty eri koordinaatistoissa, joiden muunnos ei ole häviötöntä. Niistä
+ei siis voi yksiselitteisesti koostaa koko rataverkkoa. Geometrioiden tietomalli noudattaa läheisesti lähdeaineiston
+(LandXML) käsitteistöä.
+
+### Termistö
+
+| Luokkanimi                  | Tietokantataulu                   | Käsite suomeksi                                  | Selite                                                               |
```

Comment:
Laitoin lisänä tänne termistötaulukot. En ole varma onko tästä hyötyä tai pitäisikö tällaiset vetää mielummin yhdeksi koostetuksi vai näin per-asia taulukoina. Mielipiteitä?

Reply:
Tämä on mielestäni ihan OK näinkin. Noita sekalaisia termejä on kuitenkin ainakin toistaiseksi aika vähän. Ainoa muutos minkä itse ehkä tekisin olisi, että siirtäisin tämän tuon "Tarkat geometriat" -osion alta vaikka omaksi 2. tason otsikokseen tuohon data flow -kuvan alle, koska nyt tästä rakenteesta saa sen käsitteen että täällä saattaisi olla pelkästään ihan puhtaasti geometriaan liittyviä käsitteitä

Reply:
Joo mietin sama ja se tekee vähän duplikaattia myös. Ajatuksena kuitenkin että ne näkyisi tässä koska näkyy tuossa alla olevassa kuvassa. Tietty yksi vaihtoehto olisi tehdä vain yksi isompi termistötaulukko näiden osioiden yläpuolelle.